package.json

```diff
@@ -849,6 +965,23 @@
           "description": "If enabled, dragging with the mouse activates Visual mode.",
           "default": true
         },
+        "vim.keymodel": {
+          "type": "string",
+          "markdownDescription": "Defines the behavior of shifted arrow keys and List of comma separated words, which enable special things that keys can do. These values can be used:\n\t `startsel` -> Using a shifted special key starts selection (either Select mode or Visual mode, depending on 'key' being present in 'selectmode').\n\t `stopsel` -> Using a not-shifted special key stops selection.\nSpecial keys in this context are the cursor keys, `<End>`, `<Home>`, `<PageUp>` and `<PageDown>`.",
+          "enum": [
+            "",
+            "startsel",
+            "stopsel",
+            "startsel,stopsel"
+          ],
+          "default": "startsel"
+        },
+        "vim.selectmode": {
+          "type": "string",
+          "markdownDescription": "This is a comma separated list of words, which specifies when to start Select mode instead of Visual mode, when a selection is started.\nPossible values:\n\t `mouse`\t -> when using the mouse\n\t `key`\t -> when using shifted special keys\n\t `cmd`\t -> when using 'v', 'V' or 'CTRL-V'",
+          "pattern": "^(?:mouse(?:,\\s*|\\b)|key(?:,\\s*|\\b)|cmd(?:,\\s*|\\b)){1,3}$",
```

Comment:
This pattern doesn't seem to match an empty string. I am wondering is empty string allowed, so basically I want to disable select mode.

Reply:
Nice catch! Yes empty string should be allowed!

Reply:
This is now fixed. Thank you!